### ISSUE TITLE
Freighter Base Backup/Restore

### ISSUE THREAD

**Issue by DigitalHytop** [goatfungus/NMSSaveEditor#749]
Still after all this time since many updates ago, I can no longer backup or restore my freighter's base.  Is this planned on being added back, or is this a feature we no longer have an option to take advantage of?

**Comment by goatfungus** [comment]
I'll add that back in for the next update. The base structure has changed since I last updated it.

**Comment by DigitalHytop** [comment]
> I'll add that back in for the next update. The base structure has changed since I last updated it.

Thank you Goatfungus.  I was in the middle of building a new base in creative mode and used this feature to bounce between new builds I worked on, and going back to the one I used most.  Made for building new bases a whole lot easier without losing previous builds.  Also made it easy to share those builds with friends.  Thank you too for the speedy reply.
